src/prettierRule.ts

```diff
@@ -7,10 +7,21 @@ import * as ts from 'typescript';
 
 export class Rule extends tslint.Rules.AbstractRule {
   public apply(sourceFile: ts.SourceFile): tslint.RuleFailure[] {
+    if (this.sourceFileIsIgnored(sourceFile, this.ruleArguments)) {
+      return [];
+    }
     return this.applyWithWalker(
       new Walker(sourceFile, this.ruleName, this.ruleArguments),
     );
   }
+  private sourceFileIsIgnored(sf: ts.SourceFile, ruleArguments: any[]) {
+    let ignorePath = '.prettierignore';
+    if (ruleArguments.length === 3) {
+      ignorePath = ruleArguments[2];
```

Comment:
It'd be better to set `ignorePath` from the second argument, `"prettier": [true, null, { ignorePath: ".prettierignore" }]` for example.

Reply:
no problem I can change that